diagnostic.md

```diff
@@ -3,36 +3,40 @@
 What are three advantages of using Ember?
 
 ```sh
-# your answer here
+Embers CLI, Ember usually will use much less code and look more elegant than most other
+frameworks, URL support
 ```
 
 What is the templating system used in Ember? How it is related to
 handlebars?
 
 ```sh
-# your answer here
+Templates are backed with context in both Handlebars and Ember, in Emeber the
+context is the controller
 ```
 
 Name at least two layers of an Ember application (1.0 or 2.0):
 
 ```sh
-# your answer here
+Routes and Controllers
 ```
 
 How does a component differ from an Ember template?
 
 ```sh
-# your answer here
+You can create your own tags with components
```

Comment:
Yes... I would more say that the component layer controls the data and functionality of it's corresponding template.
.5/1